CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
 ### Fixes
```

Comment:
Actually we need to document native SDK bug fixes coming from https://github.com/Microsoft/AppCenter-SDK-Apple/releases/tag/1.11.0 and https://github.com/Microsoft/AppCenter-SDK-Android/releases/tag/1.10.0 that we will integrate in this release.

Features are not applicable though.